### PR TITLE
feat: update net6

### PR DESCRIPTION
**Pull request by nreal**
# Background
Update net6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim AS restore
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS restore
```
Comment (andresmoschini) [inline Dockerfile:1]: Only curiosity. Why did you remove the distribution?

File: Doppler.Currency/Startup.cs [hunk 2]
```diff
@@ -33,8 +32,7 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers()
-                .AddJsonOptions(options => { options.JsonSerializerOptions.IgnoreNullValues = true; });
+            services.AddControllers();
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:35]: Why did you remove `IgnoreNullValues` configuration? it is by default now? are you not changing the behavior?

Comment (nreal) [reply Doppler.Currency/Startup.cs:35]: Because he is obsolete and not necessary in this API

File: Dockerfile (hunk 1 above)
Comment (nreal) [reply Dockerfile:1]: Because mcr.microsoft.com/dotnet/sdk:6.0-buster-slim does not exist in https://hub.docker.com/_/microsoft-dotnet

File: Doppler.Currency/Startup.cs (hunk 2 above)
Comment (andresmoschini) [reply Doppler.Currency/Startup.cs:35]: Is it the default behavior now?

Comment (nreal) [reply Doppler.Currency/Startup.cs:35]: Yes

Comment (nreal) [reply Doppler.Currency/Startup.cs:35]: jeje sorry without this line `options.JsonSerializerOptions.IgnoreNullValues = true;` the default behavior is `false`(change de behavior), I think that is not necessary for this API to ignore null values, but I can do a change in another pull request because that is not having a relationship with update net6

File: Dockerfile (hunk 1 above)
Comment (andresmoschini) [reply Dockerfile:1]: It is because `buster-slim` is an old distribution. I suppose that the updated equivalent wold be `bullseye-slim`.
